### PR TITLE
linked the article to pipes

### PR DESCRIPTION
**Pull request by rajdeep-biswas**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by therajdeepbiswas** [comment]
@googlebot I signed it!

File: aio/content/guide/architecture.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ There are two types of data binding:
 
 Before a view is displayed, Angular evaluates the directives and resolves the binding syntax in the template to modify the HTML elements and the DOM, according to your program data and logic. Angular supports *two-way data binding*, meaning that changes in the DOM, such as user choices, are also reflected in your program data.
 
-Your templates can use *pipes* to improve the user experience by transforming values for display.
+Your templates can use [*pipes*](https://angular.io/guide/pipes) to improve the user experience by transforming values for display.
```
Comment (gkalpak) [inline aio/content/guide/architecture.md:74]: This should be:

```suggestion
Your templates can use [*pipes*](guide/pipes) to improve the user experience by transforming values for display.
```

Would it be more appropriate to link to `guide/architecture-components#pipes` (which is also the URL we use further down in this guide)?

**Review by gkalpak — COMMENTED** [review]
Nice! Thx, @therajdeepbiswas :+1:
I've left a suggestion, but otherwise lgtm.

**Review by jessicajaniuk — APPROVED** [review]
LGTM 🍪

**Comment by jessicajaniuk** [comment]
@therajdeepbiswas Looks like you just need a `target:` label to get things merged.

File: aio/content/guide/architecture.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ There are two types of data binding:
 
 Before a view is displayed, Angular evaluates the directives and resolves the binding syntax in the template to modify the HTML elements and the DOM, according to your program data and logic. Angular supports *two-way data binding*, meaning that changes in the DOM, such as user choices, are also reflected in your program data.
 
-Your templates can use *pipes* to improve the user experience by transforming values for display.
+Your templates can use [*pipes*](guide/pipes) to improve the user experience by transforming values for display.
```
Comment (gkalpak) [inline aio/content/guide/architecture.md:74]: I still wonder if it be more appropriate to link to `guide/architecture-components#pipes` (which is also the URL we use [further down][1] in this guide). 🤔

@aikidave, @kapunahelewong: WDYT?

[1]: https://github.com/angular/angular/blame/fc80502ef706bc0e2cde560741b7fdfffb5f54cc/aio/content/guide/architecture.md#L151

**Review by gkalpak — CHANGES_REQUESTED** [review]
@therajdeepbiswas, can you please squash the two commits together and update the commit message to more closely follow our [commit message guidelines](https://github.com/angular/angular/blob/655507012b7b59c83e27429349ce3ac10608a82d/CONTRIBUTING.md#commit). For example:

> docs: add links to pipes guide in \`architecture.md\`

File: aio/content/guide/architecture.md (hunk 2 above)
Comment (kapunahelewong) [reply aio/content/guide/architecture.md:74]: Yes, I think `architecture-components#pipes` would be good here.

**Comment by AndrewKushnir** [comment]
Closing this PR due to the lack of reply from the author. Please reopen when you get a chance to look into this. Thank you.

**Pull request closed without merging**